src/govuk/components/summary-list/_index.scss

```
@@ -166,4 +166,72 @@
       }
     }
   }
+
+  // Additional block for the summary card
+  //
+  // We ship this with the summary list instead of making it it's own component
+  // because it's only interacted with via the summary list and it avoids awkwardness
+  // with trying to document a "private" component that's controlled via the
+  // component it's wrapping
+  .govuk-summary-card {
+    @include govuk-responsive-margin(6, "bottom");
+    border: 1px solid $govuk-border-colour;
+  }
+
+  .govuk-summary-card__title-wrapper {
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+    padding: govuk-spacing(3) govuk-spacing(4);
+
+    // Ensures the card header appears separate to the summary list in forced colours mode
+    border-bottom: 1px solid transparent;
+    background-color: govuk-colour("light-grey", $legacy: "grey-3");
+  }
+
+  .govuk-summary-card__title {
+    @include govuk-font($size: 19, $weight: bold);
+    margin: govuk-spacing(1) govuk-spacing(4) govuk-spacing(1) 0;
+  }
+
+  .govuk-summary-card__actions {
+    @include govuk-font($size: 19, $weight: bold);
+    display: flex;
+    flex-wrap: wrap;
+    margin: govuk-spacing(1) 0;
+    padding: 0;
+    list-style: none;
+  }
+
+  .govuk-summary-card__action {
+    display: inline;
+    margin-right: govuk-spacing(2);
+    padding-right: govuk-spacing(2);
+  }
+
+  .govuk-summary-card__action:last-child {
+    margin-right: 0;
+    padding-right: 0;
+  }
+
+  .govuk-summary-card__action:not(:last-child) {
+    border-right: 1px solid $govuk-border-colour;
+  }
```

This is really nitpicky, and happy for this to be merged as is, but I find it a little bit weird that we've got two different ways of styling the last child here which means we need both `:last-child` and `:not(:last-child)`…

- we set margin and padding on all actions and then _unset_ them on the last child
- we add the border to everything _except_ the last child

Wondering if it's clearer if we just do one of them?

```suggestion
  .govuk-summary-card__action {
    display: inline;
    margin-right: govuk-spacing(2);
    padding-right: govuk-spacing(2);
    border-right: 1px solid $govuk-border-colour;
  }

  .govuk-summary-card__action:last-child {
    margin-right: 0;
    padding-right: 0;
    border-right: none;
  }
```

(I'm suggesting this way around because otherwise there'll be no spacing between actions in IE8 which doesn't support `:not()`)

---

I'm not sure how this got here 🤔 Probs leftover prototype code, cheers for flagging.